src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonBytesMetricsReporter.java

```diff
@@ -0,0 +1,71 @@
+package io.stargate.sgv2.jsonapi.api.v1.metrics;
+
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import io.stargate.sgv2.api.common.config.MetricsConfig;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+@ApplicationScoped
+public class JsonBytesMetricsReporter {
```

Comment:
Since it's not just length (bytes), maybe name as `JsonProcessingMetricsReporter`